From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-micada__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-micada). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cephalin committed:
redirect to correct images and delete old directory

cephalin committed:
searched and found 1 other link to media file in repo

carolinacmoravia committed:
M12491: Possible incorrect string: "local and on-premises deployment scenarios"

Hi, @serinakaye, 
According to the context, it seems that it should read as "on-premises and cloud deployment scenarios." Could you help to review this and merge if agreed, please?
Many thanks in advance.

rjagiewich committed:
edit pass: azure-stack-powershell-docs

itechedit committed:
edit pass: active-directory-b2c-custom-rest-api-netfw-secure-basic

itechedit committed:
edit pass: active-directory-b2c-custom-rest-api-netfw

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-b2c-custom-rest-api-netfw-secure-basic

itechedit committed:
edit pass: active-directory-b2c-custom-rest-api-netfw

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-b2c-custom-rest-api-netfw-secure-basic

itechedit committed:
edit pass: active-directory-b2c-custom-rest-api-netfw-secure-basic

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-b2c-custom-rest-api-netfw

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into azure-stack-powershell-docs

itechedit committed:
edit pass: active-directory-b2c-custom-rest-api-netfw

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-b2c-custom-rest-api-netfw-secure-basic

itechedit committed:
removing file

rjagiewich committed:
edit pass: azure-stack-powershell-docs

rjagiewich committed:
edit pass: azure-stack-powershell-docs

rjagiewich committed:
edit pass: azure-stack-powershell-docs

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-b2c-custom-rest-api-netfw

itechedit committed:
edit pass: active-directory-b2c-custom-rest-api-netfw

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into azure-stack-powershell-docs

rjagiewich committed:
edit pass: azure-stack-powershell-docs

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-b2c-custom-rest-api-netfw-secure-basic

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw.md

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw.md

itechedit committed:
edit pass: active-directory-b2c-custom-rest-api-netfw-secure-basic

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-b2c-custom-rest-api-netfw-secure-basic

itechedit committed:
edit pass: active-directory-b2c-custom-rest-api-netfw-secure-basic

ecfan committed:
Add updated content and art

ecfan committed:
Added updated art

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw-secure-basic.md

itechedit committed:
edit pass: active-directory-b2c-custom-rest-api-netfw-secure-cert

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw-secure-cert.md

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw-secure-cert.md

ecfan committed:
More feedback updates

ecfan committed:
Acrolinx fixes

ecfan committed:
Update next topic

ecfan committed:
Cleaned up next steps topic

ecfan committed:
Tweaks

ecfan committed:
Acrolinx fixes

ecfan committed:
Update diagnostic content

ecfan committed:
Update content, fix file names for SEO, add redirects

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cfr-logic-app

rjagiewich committed:
Merge branch 'master' into azure-stack-powershell-docs

ecfan committed:
Fix broken link

ecfan committed:
Tweaks

ecfan committed:
Address Acrolinx issues

ecfan committed:
Tweaks

ecfan committed:
Tweaks

ecfan committed:
Tweaks

v-nagta committed:
USER STORY 1101059 SaaS App Tutorial: achieve3000

Commit:
task 322: add Verizon custom reports

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cfr-logic-app

ecfan committed:
Updated art with less confusing example names

ecfan committed:
Tweak structure

ecfan committed:
Structure tweak

ecfan committed:
Tweaked intent in intro

ecfan committed:
Minor intro tweak

ecfan committed:
Minor edit

ecfan committed:
Tweak intro

ecfan committed:
Clean up next steps

ecfan committed:
Tweak test section

ecfan committed:
Tweak intro

ecfan committed:
Tweak prereqs

ecfan committed:
Tweak intro

ecfan committed:
Minor tweaks

ecfan committed:
Tweak steps

Commit:
task 322: add verizon custom reports

ecfan committed:
Minor tweaks

ecfan committed:
Tweak art

ecfan committed:
Clarified note

Commit:
task 322: edit verizon custom reports

ecfan committed:
Formatting fix

ecfan committed:
Fix broken link

v-nagta committed:
USER STORY 1097506 SaaS App Tutorial: Promapp Tutorial Update

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
USER STORY 1102071 SaaS App Tutorial: TimeLive

v-nagta committed:
updated

Commit:
task 322: incorporate review comments and add report png

ecfan committed:
Updated options to clean up resources

ecfan committed:
Updated options to clean up resources

ecfan committed:
Parity edits

ecfan committed:
Fix broken link

alkohli committed:
fine tune

Commit:
task 322: add field descriptions

v-nagta committed:
updated

tamram committed:
update storage explorers page

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-1011

v-nagta committed:
updated

rjagiewich committed:
Update azure-stack-powershell-configure-quickstart.md

v-kents committed:
moved disk storage

JasonWHowell committed:
Adding HA article for PostgreSQL

JasonWHowell committed:
Adding HA article for MySQL

rothja committed:
Pulling in a SQL VM pricing guidance article applying to Linux as well

v-kents committed:
updated typos

ecfan committed:
Tweak ToC

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cfr-logic-app

ecfan committed:
Tweak title

ecfan committed:
Heading tweak

carolinacmoravia committed:
C4452: LOCBLOCKER - Extra spaces causing content blocked for localization 

Hello, @mumian, 
We need to remove extra spaces that are blocking content for localization. 
Could you help to merge this PR, please? Many thanks in advance.

Commit:
adding advisor info

Commit:
safekeeping

rjagiewich committed:
Update azure-stack-powershell-configure-quickstart.md

rjagiewich committed:
Update azure-stack-powershell-configure-quickstart.md

rjagiewich committed:
Update azure-stack-powershell-configure-quickstart.md

iainfoulds committed:
Adding CLI common management tasks. Renaming + redirecting existing PowerShell version

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cfr-logic-app

iainfoulds committed:
Acrolinx updates

cherylmc committed:
note

rjagiewich committed:
Update azure-stack-powershell-configure-quickstart.md

cherylmc committed:
Update vpn-gateway-vpn-faq.md

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into exprem1

Commit:
advisor

Commit:
fixing subnet issue

Commit:
whoops

JasonWHowell committed:
Monitoring concept articles for MySQL and PostgreSQL

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into exprem1

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into exprem1

twooley committed:
new topics for node and component replacement

twooley committed:
new topics for node and component replacement

JasonWHowell committed:
Edits to HA concepts articles

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-1011

tamram committed:
fix using statement

dominicbetts committed:
Add data flow content

rjagiewich committed:
Update azure-stack-powershell-configure-quickstart.md

YuriDio committed:
updating article

YuriDio committed:
updating article 2

carolinacmoravia committed:
CC5988: Picture description line break #45 #19814

twooley committed:
new topics for node and component replacement

JasonWHowell committed:
Update concepts-monitoring.md

iainfoulds committed:
Merge branch 'master' into vmssmanagecli

ShawnJackson committed:
Update azure-stack-powershell-configure-quickstart.md

ShawnJackson committed:
Merge pull request #25163 from rjagiewich/azure-stack-powershell-docs

Edit pass: Azure stack powershell docs

Lisaco88 committed:
Merge pull request #24325 from cephalin/redirect

redirect to correct images and delete old directory

Commit:
Merge pull request #25887 from carolinacmoravia/patch-411

CC5988: Picture description line break #45 #19814

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into exprem1

ecfan committed:
Removed en-us from link

cherylmc committed:
format

Lisaco88 committed:
Merge pull request #25782 from JasonWHowell/HA

MySQL + PostgreSQL: New article for High Availability

ecfan committed:
Readd deleted files to add myself as author

v-shils committed:
Merge pull request #25795 from rothja/patch-22

Pulling in a SQL VM pricing guidance article applying to Linux as well

BryanLa committed:
Add FAQ for cloud services

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw.md

v-shils committed:
Merge pull request #25826 from cynthn/avsets

Adding advisor info - av sets

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw.md

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into exprem1

cherylmc committed:
format

PRMerger20 committed:
Merge pull request #25893 from BryanLa/patch-5

Add FAQ for cloud services

ecfan committed:
Delete renamed topics

wmgries committed:
Update Windows page with the newest Windows Server version.

v-shils committed:
Merge pull request #25863 from iainfoulds/vmssmanagecli

VMSS common CLI management tasks

v-shils committed:
Merge pull request #25870 from JasonWHowell/OrcasMonitoring

Monitoring concept articles for MySQL and PostgreSQL

v-shils committed:
Merge pull request #25876 from dominicbetts/suite-cfbrowse

Add data flow content

PRMerger19 committed:
Merge pull request #24809 from carolinacmoravia/patch-398

M12491: Possible incorrect string: "local and on-premises deployment …

v-shils committed:
Merge pull request #25882 from YuriDio/OctoberUpdates_10

updating article

wmgries committed:
Fix minor typo.

JasonWHowell committed:
Update overview.md

wmgries committed:
Windows 10 version 1511 support expired on October 10, 2017.

wmgries committed:
Incorrect number of columns in table.

JasonWHowell committed:
Update overview.md

samedder committed:
Fixing sfctl documentation to point to latest version

JasonWHowell committed:
Update overview.md

JasonWHowell committed:
Update overview.md

Lisaco88 committed:
Merge pull request #25763 from v-kents/DiskStoragePlacement

moved disk storage

Commit:
title change to overview

v-shils committed:
Merge pull request #25262 from ecfan/cfr-logic-app

CFR updates for "Create your first logic app"

JasonWHowell committed:
Update overview.md

JasonWHowell committed:
Update overview.md

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw-secure-basic.md

twooley committed:
new topics for node and component replacement

twooley committed:
new topics for node and component replacement

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw-secure-cert.md

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw-secure-cert.md

itechedit committed:
Update active-directory-b2c-custom-rest-api-netfw-secure-cert.md

PRMerger17 committed:
Merge pull request #25822 from carolinacmoravia/patch-407

C4452: LOCBLOCKER - Extra spaces causing content blocked for localiza…

PRMerger20 committed:
Merge pull request #25904 from samedder/update_sfctl_major_version

Fixing sfctl documentation to point to latest version

GitHubber17 committed:
Merge pull request #25487 from v-nagta/achieve3000

USER STORY 1101059 SaaS App Tutorial: achieve3000

ShawnJackson committed:
Update active-directory-b2c-custom-rest-api-netfw.md

ShawnJackson committed:
Merge pull request #25148 from itechedit/active-directory-b2c-custom-rest-api-netfw-secure-basic

edit pass: active-directory-b2c-custom-rest-api-netfw-secure-basic

PRMerger12 committed:
Merge pull request #25865 from cherylmc/patch-1

Update vpn-gateway-vpn-faq.md

GitHubber17 committed:
Merge pull request #25579 from v-nagta/promappv2

USER STORY 1097506 SaaS App Tutorial: Promapp Tutorial Update

ShawnJackson committed:
Update active-directory-b2c-custom-rest-api-netfw-secure-cert.md

PRMerger14 committed:
Merge pull request #25687 from tamram/tamram-1011

update storage explorers page

ShawnJackson committed:
Merge pull request #25145 from itechedit/active-directory-b2c-custom-rest-api-netfw

edit pass: active-directory-b2c-custom-rest-api-netfw

mimig1 committed:
Cosmos DB: Corrected enclosing brackets

Lisaco88 committed:
Merge pull request #25872 from twooley/twfru

new topics for node and component replacement

ShawnJackson committed:
Merge pull request #25251 from itechedit/active-directory-b2c-custom-rest-api-netfw-secure-cert

edit pass: active-directory-b2c-custom-rest-api-netfw-secure-cert

wmgries committed:
Insert breaking character.

Lisaco88 committed:
Merge pull request #25908 from JasonWHowell/patch-3

PostgreSQL: Update overview.md

rmca14 committed:
Merge pull request #25564 from dksimpson/dean-322

task 322: add verizon custom reports

GitHubber17 committed:
Merge pull request #25592 from v-nagta/timelive

USER STORY 1102071 SaaS App Tutorial: TimeLive

PRMerger16 committed:
Merge pull request #25906 from bradsev/tdsp-cleanup-2

title change to overview

PRMerger15 committed:
Merge pull request #25665 from alkohli/SSmonitor

Added definitions for growth and usage

twooley committed:
fixing metadata

twooley committed:
fixing metadata

PRMerger19 committed:
Merge pull request #25900 from wmgries/update-windows-versions

Azure Files Windows Server semi-annual channel Support

Lisaco88 committed:
Merge pull request #25864 from cherylmc/exprem1

note

Commit:
Merge pull request #25920 from mimig1/patch-146

Cosmos DB: Corrected enclosing brackets

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Merge pull request #25928 from twooley/master

Fixed metadata and added note about maintenance window

v-alje committed:
Merge pull request #25929 from MicrosoftDocs/FromPublicRepo

From public repo

Lisaco88 committed:
Merge pull request #25902 from JasonWHowell/patch-1

MySQL: Update overview.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
